tests/Test/src/test_z.birch

```diff
@@ -1,30 +1,32 @@
 /*
  * Test the normalizing constant of a pdf.
  *
- * - π: The target distribution. 
- * - N: Number of importance samples.
- * - lazy: Use lazy expressions?
+ * @param π The target distribution. 
```

Comment:
Nice, thanks for updating, did not realize that I'd neglected to update the `Test` package with the new Doxygen-style commands!